marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/jobs/job-query-lib.sjs

```diff
@@ -17,26 +17,26 @@
 
 const op = require('/MarkLogic/optic');
 
-function getJobData(start, pageLength, sortColumn, sortDirection) {
+function findStepResponses(start, pageLength, sortColumn, sortDirection) {
   let orderByConstraint = op.desc('startTime');
   if(sortColumn) {
     orderByConstraint = sortDirection === 'ascending' ? op.asc(sortColumn) : op.desc(sortColumn);
   }
 
   const response = {};
-  const totalCountQuery = 'select count(*) as total from Job.jobMonitor';
-  const jobsDataQuery = 'select Job.jobMonitor.stepName as stepName,' +
-      'Job.jobMonitor.stepType as stepType,' +
-      'Job.jobMonitor.stepStatus as status,' +
-      'Job.jobMonitor.targetEntityType as entityName,' +
-      'Job.jobMonitor.stepStartTime as startTime,' +
-      'Job.jobMonitor.stepEndTime - Job.jobMonitor.stepStartTime as duration,' +
-      'Job.jobMonitor.successfulEvents as successfulEvents,' +
-      'Job.jobMonitor.failedEvents as failedEvents,' +
-      'Job.jobMonitor.userId as userId,' +
-      'Job.jobMonitor.jobId as jobId,' +
-      'Job.jobMonitor.flowName as flowName' +
-      ' from Job.jobMonitor';
+  const totalCountQuery = 'select count(*) as total from Job.StepResponse';
+  const jobsDataQuery = 'select Job.StepResponse.stepName as stepName,' +
+      'Job.StepResponse.stepType as stepDefinitionType,' +
+      'Job.StepResponse.stepStatus as status,' +
+      'Job.StepResponse.targetEntityType as entityName,' +
+      'Job.StepResponse.stepStartTime as startTime,' +
+      'Job.StepResponse.stepEndTime - Job.StepResponse.stepStartTime as duration,' +
+      'Job.StepResponse.successfulEvents as successfulItemCount,' +
```

Comment:
I think the template needs to be updated with these names also - the template is projecting column names on top of the data, where we can rename things that have misleading names.